routers/api/v1/utils/hook.go

```diff
@@ -87,6 +87,14 @@ func AddRepoHook(ctx *context.APIContext, form *api.CreateHookOption) {
 	}
 }
 
+func issuesHook(events []string, event string) bool {
+	return com.IsSliceContainsStr(events, event) || com.IsSliceContainsStr(events, string(models.HookEventIssues))
+}
+
+func pullHook(events []string, event string) bool {
+	return com.IsSliceContainsStr(events, event) || com.IsSliceContainsStr(events, string(models.HookEventPullRequest))
+}
+
```

Comment:
This is my attempt to making the API backwards compatible. When creating a webhook via API, `issues` will enable _all_ issue events and `pull_request` similarly.